spec/controllers/api/v1/workflow_contents_controller_spec.rb

```diff
@@ -22,6 +15,14 @@
   let(:primary_content) { workflow.primary_content }
 
   describe "#index" do
+    let!(:acl) do
```

Comment:
Yeah I'd have to either extract them and have a conditional not create them for the #destroy tests or extract them into a method a have a before(:each) call that method in each of the describe blocks. I neither one really seems better to me.